geode-web/src/test/java/org/apache/geode/management/internal/cli/commands/ConfigCommandsDUnitTest.java

```diff
@@ -52,7 +52,7 @@
 
 @Category(DistributedTest.class)
 @RunWith(JUnitParamsRunner.class)
-public class ConfigCommandDUnitTest {
+public class ConfigCommandsDUnitTest {
```

Comment:
I would recommend rip out the alterRuntime related tests out of this class and name it AlterRuntime..Test. This way, we know AlterRuntimeCommand has corresponding tests instead of trying to find the tests inside ConfigCommandDUnitTest

Reply:
I had thought about pulling the alter runtime tests out into a new class, but ended up leaving everything lumped together. I'll go ahead and make the separation ...

Reply:
Split out 'alter runtime'  tests into their own class

Reply:
Would you mind separating what's left of ConfigCommandsDUnitTest into DescribeConfigCommandDUnitTest and ExportConfigCommandDUnitTest?  I think that would also make the tests easier to discover.

Reply:
OK. Pushed another commit with describe config and export config command test in their own classes